File: api/src/models/Ratings.js
const mongoose = require("mongoose");

const Favorites = mongoose.model("Favorites", {
    movies: {type: mongoose.ObjectId, ref: 'Movies', required: true},
    users: {type: mongoose.ObjectId, ref: 'Users', required: true},
    rating: {type: Number, required: true}
});

module.exports = Movies;

File: app_mobile/lib/models/ratings.dart
class Rating {
  final String id;
  final String movies;
  final String users;
  final int rating;

  Rating({
    required this.id,
    required this.movies,
    required this.users,
    required this.rating,
  });

  factory Rating.fromJson(Map<String, dynamic> json) {
    return Rating(
      id: json['_id'],
      movies: json['movies'],
      users: json['users'],
      rating: json['rating'],
    );
  }
}
